src/Symfony/Component/ErrorHandler/Attribute/LogLevel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Attribute;
+
+/**
+ * @author Dejan Angelov <angelovdejan@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class LogLevel
+{
+    public function __construct(public readonly string $level)
+    {
+        if (!defined(sprintf("\Psr\Log\LogLevel::%s", strtoupper($this->level)))) {
+            throw new \InvalidArgumentException(sprintf("Invalid log level - \"%s\".", $this->level));
+        }
```

Comment:
Using an enum would definitely look better here IMO, but I thought it'd be better to reuse the existing list from the PSR package that we're already using, instead of copying the list of levels.